R/timeseriessummary-data.R

```diff
@@ -217,7 +218,7 @@ constructTSDetails <- function(reportData, timezone){
 #' @param timezone the timezone of the report
 parseTSSThresholds <- function(reportData, timezone){
   thresholds <- tryCatch({
-    readThresholds(reportData)
+    readTSSThresholds(reportData)
```

Comment:
this was using the generic readThresholds and since the fields changed and CORR was also using it, I had to make its own function. I will update the CORR tickets to suggest re-merging them when that's also transitioned.

Reply:
https://internal.cida.usgs.gov/jira/browse/AQCU-1539?focusedCommentId=140700&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-140700